UnattendResources/Logon.ps1

```diff
@@ -228,6 +260,14 @@ try
     $Host.UI.RawUI.WindowTitle = "Running SetSetupComplete..."
     & "$programFilesDir\Cloudbase Solutions\Cloudbase-Init\bin\SetSetupComplete.cmd"
 
+    #Apply local security settings
+     
+    $Host.UI.RawUI.WindowTitle = "Applying Local Security setting..."

```

Comment:
The policies will be applied through GPO once the machine is added to Domain, however this will be helpful for a standalone machine which has to be compliant as per organization security standards. This is required in our scenario.